src/main/java/Task.java

```diff
@@ -0,0 +1,21 @@
+public class Task {
+    protected String taskDescription;
+    protected boolean isDone;
+
+    public Task(String taskDescription) {
+        this.taskDescription = taskDescription;
+        this.isDone = false;
+    }
+
+    public boolean isDone() {
+        return isDone;
+    }
+
+    public void setDone(boolean done) {
+        isDone = done;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); //Marks a task done with an X
```

Comment:
Short comments can appear on the same line as the code they describe, but should be shifted far enough to separate them from the statements. Alternatively, you may consider adding the comment on a separate line, indented relative to their position in the code.